model/Service/TimeoutCommand.php

```
@@ -35,23 +36,12 @@ final class TimeoutCommand implements
-
-    /** @var bool */
-    private $lateSubmissionAllowed;
-
     public function __construct(
-        QtiRunnerServiceContext $serviceContext,
-        bool $hasStartTimer,
-        bool $lateSubmissionAllowed
+        private QtiRunnerServiceContext $serviceContext,
+        private FeatureFlagCheckerInterface $featureFlagChecker,
+        private bool $hasStartTimer,
+        private bool $lateSubmissionAllowed
```

PHPMD (2.15.0)

Avoid excessively long variable names like $lateSubmissionAllowed. Keep variable name length under 20. (Naming Rules)

(LongVariable)